release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -4039,4 +4039,23 @@ For more information about logging levels for MetalLB, see xref:../networking/me
 [id="ocp-4-10-60-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
```

Comment:
This like nitpick goes back a few releases and did not originate with your PR, but we cannot determine when the module name was changed. Perhaps from this release and onwards we can make the following change?

Updating a cluster within a minor version by using the CLI > Updating a cluster using the CLI

Reply:
You all might want to consider opening a ticket to investigate this xref further and update all of them so the links are consistent throughout the RNs.

Reply:
I can take this. I'll correct these in future z-stream releases. Thanks, Olivia.